### ISSUE TITLE
feature request: external possibility to save the state of a running calculation

### ISSUE THREAD

**Issue by SpiritSuperUser** [spirit-code/spirit#250]
it would be create to have an external possibility to save the current state of a running calculation. This could be done with an external file like the "STOP" file which triggers a running calculation to write the current image(s), energies etc for the current iteration

**Comment by GPMueller** [comment]
A STOP file will cause the current simulation to stop, triggering it to write the output corresponding to the settings. You could e.g. set `llg_output_final` to on but most other settings to off in order to only write output when you create a STOP file.
Does this correspond to what you want to do?

**Comment by SpiritSuperUser** [comment]
no, I do not want to stop the running calculation, I just want to check the running calculation. E.g. when I have a calculation running, where only `llg_output_final` is set to on, I do not get any output during the calculation. This is fine and the wanted behavior, but sometimes it might be useful to check a running calculation for example because it does not converge or converges just very slowly. In those cases it would be great to tell spirit, that one wants to get the current state, but without stopping the calculation itself. One could of course create the STOP file, check the calculation, and start the calculation than again with the written output, but this is not very convenient.

so in conclusion, I would like to create e.g. a PRINTOUT file, which triggers the writing of output files of the current simulation without stopping the simulation

**Comment by GPMueller** [comment]
This is an interesting idea and may indeed be useful on clusters, I agree this should be added.
